package.json

```diff
@@ -62,6 +62,7 @@
   "dependencies": {
     "invariant": "^2.2.1",
     "lodash": "^4.13.1",
```

Comment:
Yes. lodash will still be used in the source code. The only piece that changes is the `es` compiled files.These distribution files are rewritten to utilize lodash-es instead of lodash.